.travis.yml

```diff
@@ -10,6 +10,8 @@ matrix:
   - jdk: openjdk7
   - jdk: openjdk8
   - jdk: oraclejdk8
+  - jdk: openjdk11
+  - jdk: oraclejdk11
```

Comment:
See my comment on your other PR about this: https://github.com/diffblue/deeptest-utils/pull/68